### PR TITLE
Add Frontify tab to the Experience Fragment editor

### LINKED ISSUE #16: Frontify tab is not available to edit Experience Fragments

### PR DESCRIPTION
**Pull request by ValtechDeBruyneYves**
closes #16

### CODE REVIEW

File: ui.apps/src/main/content/META-INF/vault/filter.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <workspaceFilter version="1.0">
     <filter root="/apps/frontify-aem-connector"/>
     <filter root="/apps/wcm" mode="merge"/>
+    <filter root="/apps/cq" mode="merge"/>
```
Comment (nhirrle) [inline ui.apps/src/main/content/META-INF/vault/filter.xml:5]: You may want to limit this to /apps/cq/experience-fragments/content/editor.

Comment (nhirrle) [inline ui.apps/src/main/content/META-INF/vault/filter.xml:5]: You may want to limit this to /apps/cq/experience-fragments/content/editor

**Review by nhirrle — COMMENTED** [review]
PR looks fine, just 1 non critical comment left

File: ui.apps/src/main/content/META-INF/vault/filter.xml (hunk 1 above)
Comment (ValtechDeBruyneYves) [reply ui.apps/src/main/content/META-INF/vault/filter.xml:5]: I could update the filter path to /apps/cq/experience-fragments

Any more entries in the path result in: 
Filter root's ancestor '/apps/cq/experience-fragments' is not covered by any of the specified dependencies.

**Pull request merged**